### PR TITLE
perf(annotate): 优化代码注释内容

### PR DESCRIPTION
**Pull request by lbbniu**
**Please provide issue(s) of this PR:**
Fixes #

**To help us figure out who should review this PR, please put an X in all the areas that this PR affects.**

- [ ] Configuration
- [ ] Docs
- [ ] Installation
- [ ] Performance and Scalability
- [x] Naming
- [ ] HealthCheck
- [ ] Test and Release

**Please check any characteristics that apply to this pull request.**

- [x] Does not have any user-facing changes. This may include API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

**Review by onecer — COMMENTED** [review]
LGTM

File: cmd/start.go [hunk 1]
```diff
@@ -38,5 +38,6 @@ var (
 
 // init 解析命令参数
 func init() {
+	startCmd.InheritedFlags()
```
Comment (andrewshan) [inline cmd/start.go:41]: 这个InheritedFlags的作用是什么？

Comment (chuntaojun) [reply cmd/start.go:41]: returns all flags which were inherited from parent commands.

Comment (lbbniu) [reply cmd/start.go:41]: 阅读源码，误提交，已回滚修改

**Comment by chuntaojun** [comment]
pls fix ci

**Comment by lbbniu** [comment]
> pls fix ci

pls run ci

**Pull request merged**